ui/desktop/tests/acceptance/scopes-test.js

```diff
@@ -282,53 +282,4 @@ module('Acceptance | scopes', function (hooks) {
       'Cannot retry'
     );
   });
-
-  test('handles cli error on connect', async function (assert) {
```

Comment:
Are these tests not valid anymore, shouldn't we still be testing these scenarios?

Can we actually add a few tests for testing bare bones permissions as well?

Reply:
These tests are duplicates of exactly what we are doing in the `target-test.js` file. Plus we used to be able to connect from this screen, but that is not true anymore. It all happens on the target-details route now.